### PR TITLE
Added DZ8.Spring MVC and DZ10.Moving hibernate.config to class

### PR DESCRIPTION
**Pull request by VladyslavSolo**

### CODE REVIEW

File: Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/AccountController.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    public String getAuthorization() {
+        return "Authorization";
+    }
+
+    @RequestMapping(method = POST, value = "/auth")
+    public String postAuthorization(@RequestParam("login") String login,
+                                    @RequestParam("password") String password,
+                                    HttpSession session) {
+        log.info("Current login:" + login + " Current password:" + password);
+        if ((login != null || password != null) && (userService.authorizate(login,password))) {
```
Comment (dinamio) [inline Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/AccountController.java:39]: Authorizate is not appropriate name. Login and password null checks are wrong

File: Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/QuestionController.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    @Autowired
+    QuizDAOReal quizDAO;
+    @Autowired
+    QuizService quizService;
+    @Autowired
+    QuestionDAOReal questionDAO;
+
+    @RequestMapping(method = GET, value = "questions/{id}")
+    public String getFillQuestionList(Model model, @PathVariable("id") Integer id){
```
Comment (dinamio) [inline Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/QuestionController.java:35]: Naming convention?

File: Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/service/UserService.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        if (currentUser!= null && pass.equals(currentUser.getPassword())) return true;
+        return false;
+    }
+    public boolean authorizate(User user) {
+        currentUser = userDAOReal.findByLogin(user.getLogin());
+        if (currentUser!= null && user.getPassword().equals(currentUser.getPassword())) return true;
+        return false;
+    }
+
+    public User getCurrentUser() {
```
Comment (dinamio) [inline Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/service/UserService.java:35]: This not the field of the service.

**Review by dinamio — APPROVED** [review]
Change approach of current user and current quiz

File: Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/AccountController.java (hunk 1 above)
Comment (VladyslavSolo) [reply Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/AccountController.java:39]: first logical operation will be deleted. 
Can it be possible name to method for example: "isRightAuthData" ?

File: Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/QuestionController.java (hunk 2 above)
Comment (VladyslavSolo) [reply Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/controllers/QuestionController.java:35]: in the next pull request will be fixed

**Pull request merged**